deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -2,6 +2,29 @@
 from deepchem.utils.molecule_feature_utils import one_hot_encode
 from deepchem.utils.typing import RDKitMol, RDKitAtom
 import numpy as np
+from typing import Tuple, Any
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
These changes are from the other MATFeaturizer PR right? Can you remove the changes from this PR to make it easier for me to review?

Reply:
I have removed changes in this and `test_mat_featurizer.py` as well from this PR now.